changelogs/CHANGELOG-v1.23.0.md

```diff
@@ -0,0 +1,110 @@
+For more information [see here](https://projectcontour.io/docs/main/config/slow-start/).
+
+(#4772, @tsaarni)
+
+
+# Minor Changes
+
+## HTTPProxy CORS policy supports regex matching on Allowed Origins
+
+The AllowOrigin field of the HTTPProxy CORSPolicy can be configured as a regex to enable more flexibility for users.
+More advanced matching can now be performed on the `Origin` header of HTTP requests, instead of restricting users to allow all origins, or enumerating all possible values.
+
+(#4710, @sunjayBhatia)
+
+
+# Other Changes
+- Transition to `default_source_code` Lua filter field from deprecated `inline_string` field for specifying Lua scripts. (#4622, @sunjayBhatia)
+- There are so many EnsureXDeleted in the sub-packages of objects , so unify them to objects/EnsureObjectDelete (#4630, @izturn)
+- Transition to using new bootstrap field `default_regex_engine` instead of deprecated per-regex match engine selection. (#4652, @sunjayBhatia)
+- Gateway Listeners with Secret references whose namespace is not covered by a ReferenceGrant should have their status reason set to RefNotPermitted. (#4664, @sunjayBhatia)
+- Add a new flag `leader-election-namespace` for gateway-provisioner (#4669, @izturn)
+- Add Contour log level configurability to ContourDeployment resource. (#4676, @izturn)
+- Add Kubernetes client debug log level configurability to ContourDeployment resource. (#4677, @izturn)
+- add the fields extraVolumes & extraVolumeMounts to crd/ContourDeployment to enable Envoy pods to mount additional volumes (#4680, @izturn)
+- Add Kubernetes annotations configurability to ContourDeployment resource. to enable customize pod annotations for pod/envoy (#4681, @izturn)
+- Add Kubernetes resource labels configurability to ContourDeployment resource. (#4709, @izturn)
+- Add resource requirements configurability to ContourDeployment to enable resource quota for containers. (#4712, @izturn)
+- Gateway API: status-only updates to resources no longer trigger DAG reprocessing and xDS updates. (#4744, @skriss)
+- Gateway API: don't make status update calls to the API server if status has not changed on the resource. (#4745, @skriss)
+- Updates to Gateway API v0.5.1. (#4755, @skriss)
+- Update supported Kubernetes versions to 1.23, 1.24, and 1.25. (#4757, @sunjayBhatia)
+- For Gateway API conformance, when a HTTP request matches multiple rules within a HTTPRoute, precedence is given to the rule that comes first in that HTTPRoute (in list-order). (#4763, @sunjayBhatia)
+- Updates Go to 1.19.2, see [release notes here](https://go.dev/doc/devel/release#go1.19.minor). (#4773, @sunjayBhatia)
+- Update Envoy to v1.24.0. See the [Envoy release notes](https://www.envoyproxy.io/docs/envoy/v1.24.0/version_history/v1.24/v1.24.0) for more information. (#4804, @skriss)
+
+
+# Docs Changes
+- Added guide for configuring gRPC routes. (#4725, @sunjayBhatia)
+
+
+# Deprecation and Removal Notices
+
+## Contour v1.20 minor release now out of support
```

Comment:
note this addition in particular

Reply:
Nice, we should figure out how to add this to the template